### DISCUSSION TITLE
v19.32.6-4.0.1 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#338] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/Balackburn/YTLitePlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.32.6`  
Current YTLite Version: `4.0.1`  
Current App Compatibility: `iOS/iPadOS 15.0` or later

**RELEASE 19.32.6-4.0.1:** (2024-08-14)

- Updated Youtube from `19.29.1` to `19.32.6`

## New Features
- Added YouTimeStamp (@arichornloverALT)
- Added "Copy Settings" and "Paste Settings" (@arichornlover)
- Add reporting option (@bhackel)
- Live Chat OLED dark mode (@bhackel)
- Add Hide Speed Toast (@bhackel)

## Bug Fixes
- Fix building on iOS 14 (@bhackel) [#278](https://github.com/YTLitePlus/YTLitePlus/pull/278)
- Fix dylib injection issue (@bhackel)
- Fix YouTube Extensions bug (@bhackel)
- Fix delete workflows (@bhackel)
- Fix localization and copy/paste switch (@bhackel)
- Fix LowContrastMode description (@arichornlover)
- Fix build error (@bhackel)
- Fix OLED live chat (@bhackel) [#331](https://github.com/YTLitePlus/YTLitePlus/pull/331)

## Improvements
- Cleanup Makefile and actions script (@bhackel)
- Default disable big mini player (@bhackel)
- Hide comment previews (@bhackel)
- Hide autoplay mini preview (@bhackel)
- Remove YTLitePlus OLED Theme (@bhackel)
- Copy Hide HUD and Hide Collapse Button (@bhackel)

## Localization
- Fix import/export settings localization (@bhackel)
- Turkish: Added proper localization (@gototheskinny) [#283](https://github.com/YTLitePlus/YTLitePlus/pull/283), updated (@gototheskinny) [#336](https://github.com/YTLitePlus/YTLitePlus/pull/336)
- Bulgarian: Added translations (@schweppes-0x) [#332](https://github.com/YTLitePlus/YTLitePlus/pull/332)
- Portuguese: Updated localization (@VictorxD70) [#326](https://github.com/YTLitePlus/YTLitePlus/pull/326)
- Added missing strings to localization (@Balackburn)
- Updated Localizable.strings (@manhchuduc) [#305](https://github.com/YTLitePlus/YTLitePlus/pull/305)

## Updated Submodules
- Added YouTimeStamp submodule
- YouTimeStamp
- DontEatMyContent
- YouGroupSettings
- YouTubeHeader
- iSponsorBlock
- YTABConfig
- YTUHD
- Return-YouTube-Dislikes
- YouMute
- protobuf 

## Contributors
- @bhackel
- @arichornlover
- @gototheskinny
- @Balackburn
- @manhchuduc
- @VictorxD70
- @schweppes-0x

### Issues in this Release:
- LowContrastMode does not work *(Compatible only with YouTube v17.33.2-v17.38.10)*

### Catbox
`98ns5s.ipa`

<hr /><em>This discussion was created from the release <a href='https://github.com/YTLitePlus/YTLitePlus/releases/tag/v19.32.6-4.0.1'>v19.32.6-4.0.1 - YTLitePlus</a>.</em>

**Comment by plus024** [comment]
I don't think that Github IPA is supposed to be there

**Reply by aricloverEXTRA** [reply]
@Balackburn there is an IPA in this release! I highly recommend removing it now! I can’t do it because GitHub signed me out of my other account.

**Reply by Balackburn** [reply]
> @Balackburn there is an IPA in this release! I highly recommend removing it now! I can’t do it because GitHub signed me out of my other account.

My bad

**Reply by Milchkarl** [reply]
Damn i was happy af when the mail said that there was an ipa included in the release

**Reply by JudeTheTechSavvyDude** [reply]
I've figured out a way to do the ipa using 7-Zip, via the actions, and choosing the latest release, and turning the zip into an actual ipa then unzipping the ipa file, inside of it is a another ipa file, that what i've been doing since the DMCA, it's been tested and proved by me, and it works pretty damn good with no issues, i also bet that other people tried this method before, maybe not

**Reply by mrld341** [reply]
> I've figured out a way to do the ipa using 7-Zip, via the actions, and choosing the latest release, and turning the zip into an actual ipa then unzipping the ipa file, inside of it is a another ipa file, that what i've been doing since the DMCA, it's been tested and proved by me, and it works pretty damn good with no issues, i also bet that other people tried this method before, maybe not

Ok after very long time I think I got it.
But will that be something that will get deleted eventually?

**Reply by bhackel** [reply]
You can join the uYouEnhanced telegram group for help with building. There’s over 700 people in it now: https://t.me/uYouEnhanced

**Reply by Balackburn** [reply]
> I've figured out a way to do the ipa using 7-Zip, via the actions, and choosing the latest release, and turning the zip into an actual ipa then unzipping the ipa file, inside of it is a another ipa file, that what i've been doing since the DMCA, it's been tested and proved by me, and it works pretty damn good with no issues, i also bet that other people tried this method before, maybe not

It doesn't make sense - you know that .ipa files are the same as .zip files, but with a different extension? Once you've built YTLitePlus, you download the .zip and extract it, there's an .ipa inside, no need to rename anything.

**Reply by JudeTheTechSavvyDude** [reply]
Originally the file is actually a zip file, and i convert the .zip to a .ipa by changing the format of it before i download it, then i unzip the file, because the ipa won't install and give me an error

**Reply by Balackburn** [reply]
> Originally the file is actually a zip file, and i convert the .zip to a .ipa by changing the format of it before i download it, then i unzip the file, because the ipa won't install and give me an error

Why do you rename the .zip file into a .ipa file? Just extract the .zip their is the .ipa inside, what you are doing is useless

**Comment by sujithsugadathan** [comment]
Does it crashes when playing shorts videos or is it only happens to me ?

**Reply by schweppes-0x** [reply]
same
here, when opening the shorts it crashes

**Reply by Balackburn** [reply]
> Does it crashes when playing shorts videos or is it only happens to me ?

Yes use older versions for now, as I disable shorts myself I though this version was stable.
